### PR TITLE
Feat/search userprofile

### PR DESCRIPTION
**Pull request by PeterDMadrid**
**OVERVIEW**

- Re-added search input in large screens in the navbar.

**SNIPPETS**

image
image

### CODE REVIEW

File: src/app/Http/Requests/SearchRequest.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
```
Comment (hipe01mark) [inline src/app/Http/Requests/SearchRequest.php]: remove! return type in the docbloc

File: src/app/Services/UserService.php [hunk 2]
```diff
@@ -95,4 +95,16 @@ public function getUserProfile(int $id): array
             return ['error' => 'An error occurred: ' . $e->getMessage()];
         }
     }
+
+    /**
+     * Get query
+     */
+    public function getQuery($request, int $id)
```
Comment (hipe01mark) [inline src/app/Services/UserService.php]: parameter type of $request and return type as well

Comment (PeterDMadrid) [reply src/app/Services/UserService.php]: Done

File: src/app/Http/Requests/SearchRequest.php (hunk 1 above)
Comment (PeterDMadrid) [reply src/app/Http/Requests/SearchRequest.php]: Done

File: src/app/Http/Controllers/UserController.php [hunk 3]
```diff
@@ -23,4 +22,14 @@ public function show(int $id): View
         $data = $this->userService->getUserProfile($id);
         return view('pages.profile.profile', $data);
     }
+
+    /**
+     * Show search results
+     */
+
```
Comment (hipe01mark) [inline src/app/Http/Controllers/UserController.php]: remove breakline

Comment (PeterDMadrid) [reply src/app/Http/Controllers/UserController.php]: Done

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Pull request merged**